code/datums/ai/generic_actions.dm

```diff
@@ -22,3 +22,42 @@
 /datum/ai_behavior/move_to_target/perform(delta_time, datum/ai_controller/controller)
 	. = ..()
 	finish_action(controller, TRUE)
+
+
+/datum/ai_behavior/break_spine
+	behavior_flags = AI_BEHAVIOR_REQUIRE_MOVEMENT
+	action_cooldown = 0.7 SECONDS
+	var/target_key
+	var/give_up_distance = 10
+
+
+/datum/ai_behavior/break_spine/perform(delta_time, datum/ai_controller/controller)
+	var/mob/living/batman = controller.blackboard[target_key]
+	var/mob/living/big_guy = controller.pawn //he was molded by the darkness
+
+	if(batman.stat)
+		finish_action(controller, TRUE)
+
+	if(get_dist(batman, big_guy) >= give_up_distance)
+		finish_action(controller, FALSE)
+
+	big_guy.start_pulling(batman)
+	big_guy.setDir(get_dir(big_guy, batman))
+
+	batman.visible_message("<span class='warning'>[batman] gets a slightly too tight hug from [big_guy]!</span>", "<span class='userdanger'>You feel your bones break as [big_guy] embraces you!</span>")
```

Comment:
what if they don't have bones, due to being a slimeperson or the like?

you can check for bones by checking which kinds of wounds they can receive, btw